Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
That works for me; it's concise, correct, and contains all the necessary information.